test/functional/RecursiveRecursiveIteratorLocatorTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace RebelCode\Modular\Locator\FuncTest;
+
+use Dhii\Validation\ValidatorInterface;
+use RebelCode\Modular\Locator\RecursiveIteratorLocator;
+use RecursiveArrayIterator;
+use Xpmock\TestCase;
+
+/**
+ * Tests {@see RebelCode\Modular\Locator\IteratorLocator}.
+ *
+ * @since [*next-version*]
+ */
+class RecursiveIteratorLocatorTest extends TestCase
```

Comment:
Add test for constructor, to make sure that the things are actually getting set.